src/www/ui/change-license-processPost.php

```diff
@@ -95,20 +95,31 @@ function Output()
     $decisionMark = @$_POST['decisionMark'];
     if(!empty($decisionMark))
     {
-      $itemTableName = $this->uploadDao->getUploadtreeTableName($itemId);
-      /** @var ItemTreeBounds */
-      $itemTreeBounds = $this->uploadDao->getItemTreeBounds($itemId,$itemTableName);
-      $errMsg = $this->clearingDao->markDirectoryAsIrrelevant($itemTreeBounds,$groupId,$userId);
-      if (empty($errMsg))
-      {
-        return new JsonResponse(array('result'=>'success'));
+      if(!is_array($itemId)) {
+        $this->doMarkIrrelevant($itemId, $groupId, $userId);
+      }else{
+        foreach($itemId as $uploadTreeId){
+          $this->doMarkIrrelevant($uploadTreeId, $groupId, $userId);        
```

Comment:
Here you throw away the error message

Reply:
@this point there wont be any errors as it returns directly from this doMarkIrrelevant function.

Reply:
https://github.com/siemens/fossology/blob/fcab67c6ea5b6c6caf44aaf8b1d97c24a094365e/src/www/ui/change-license-processPost.php#L174

Reply:
I do not understand what you mean.

Your function `doMarkIrrelevant` returns something related to the error message, but that is dropped and ignored both times when you call it.